include/glow/Backends/BackendOptions.h

```diff
@@ -16,7 +16,19 @@
 #ifndef GLOW_BACKENDS_BACKENDOPTIONS_H
 #define GLOW_BACKENDS_BACKENDOPTIONS_H
 
+#include "llvm/ADT/SmallVector.h"
+
 namespace glow {
+class Storage;
+
+/// Hints provided to the Backend, the backend is not required to honor them.
+struct BackendHints {
+  /// Number of execution units to reserve, 0 for unspecified.
+  unsigned executionUnits{0};
+
+  /// Storage nodes to be pinned to SRAM listed in order of priority.
+  llvm::SmallVector<Storage *, 0> SRAMPrioritization;
```

Comment:
Is there a reason for using an N=0 SmallVector?  Seems like that's not really different from a `std::vector`.